groundstation/static/js/components/AddUser.jsx

```diff
@@ -0,0 +1,209 @@
+            <Paper className="grid-containers adduser-container">
+                <div>
+                    <div style={{padding: "20px", textAlign: "center"}}>
+                        <Typography variant="h4" style={{color: "#28324C"}}>
+                            Register New User
+                        </Typography>
+                    </div>
+                    
+                    <div style={{textAlign: "center"}}>
+                        <TextField
+                            style={{width: "60%"}}
+                            required
+                            id="outlined-required"
+                            label="Username"
+                            name="username"
+                            margin="normal"
+                            variant="outlined"
+                            value={this.state.username}
+                            onChange={(event) => this.handleUsernameChange(event)}
+                            error={!(this.state.error_message === '')}
+                        />
+                    </div>
+                    <div style={{textAlign: "center"}}>
+                        <TextField
+                            style={{width: "60%"}}
+                            required
+                            id="outlined-password-input"
+                            label="Password"
+                            type="password"
+                            name="password"
+                            margin="normal"
+                            variant="outlined"
+                            value={this.state.password}
+                            onChange={(event) => this.handlePasswordChange(event)}
+                            error={!(this.state.error_message === '')}
+                        />
+                    </div>
+                    <div style={{textAlign: "center"}}>
+                        <TextField
+                            style={{width: "60%"}}
+                            required
+                            id="outlined-confirmpassword-input"
+                            label="Confirm Password"
+                            type="password"
+                            name="password"
+                            margin="normal"
+                            variant="outlined"
+                            value={this.state.password2}
+                            onChange={(event) => this.handlePassword2Change(event)}
+                            error={!(this.state.error_message === '')}
+                            onKeyDown={ (event) => this.handleKeyPress(event)}
+                        />
+                    </div>
+                    <div>
+                        {this.handleError()}
+                    </div>
+                    <div style={{textAlign: "center"}}>
+                        <FormControlLabel label="Admin User"
+                            control={<Checkbox checked={this.state.newUserIsAdmin} onChange={(event) => this.handleChecked(event)} />} />
```

Comment:
Can you add a [Tooltip](https://mui.com/material-ui/react-tooltip/) to the admin user checkbox warning users that:
- Admin users cannot be deleted
- Can change automation of commands on passovers
- Can dequeue any command from being sent
- Can create new users